src/MemoryTracing/MemoryTracingUtils.h

```diff
@@ -22,8 +22,10 @@ namespace orbit_memory_tracing {
     uint32_t pid) noexcept;
 
 [[nodiscard]] std::string GetProcessMemoryCGroupName(std::string_view cgroup_content);
-[[nodiscard]] int64_t GetCGroupMemoryLimitInBytes(std::string_view memory_limit_in_bytes_content);
-[[nodiscard]] int64_t GetRssFromCGroupMemoryStat(std::string_view memory_stat_content);
+void GetCGroupMemoryLimitInBytes(std::string_view memory_limit_in_bytes_content,
```

Comment:
I'm not against this change (actually it makes sense), but I would like to have @ronaldfw opinion here (about having a void Get method with the only returning value in the parameters).

Reply:
Thanks for the suggestion!

Fields in `CGroupMemoryUsage` are collected from different files. I changed this as a void Get method in order to keep consistent with the `GetValuesFromCGroupMemoryStat` method. But I'm also fine with returning the value.

Reply:
Thanks for the suggestion!

Fields in `CGroupMemoryUsage` are collected from different files. I changed this as a void Get method in order to keep consistent with the `GetValuesFromCGroupMemoryStat` method. But I'm also fine with returning the value.

Reply:
I think it's fine for consistency with the rest. In Google code style, parameters passed as * are in/out parameters, so it's pretty obvious that the value is returned as part of the proto that is passed in via pointer. 

You _could_ consider a different name (not Get), maybe something like `InitializeCGroupMemoryLimitInBytes`, but I don't feel too strongly about this (and I'm not sure what the best name here is).

Reply:
Thanks! I renamed these GetValues methods with the format Update<proto name>From<file name>, for instance, `UpdateSystemMemoryUsageFromMemInfo`, `UpdateCgroupMemoryUsageFromMemoryStat`.

Reply:
I really like the new names, looks great!